bibo/internals.py

```diff
@@ -111,8 +111,10 @@ def set_file(data, entry, file_, destination=None, no_copy=False):
     basename = string_to_basename(entry['key'])
     path = os.path.join(destination, basename + file_extension)
     entry['fields']['file'] = path
-    click.echo('Copying {} to {}'.format(file_, path))
-    shutil.copy(file_, path)
+    # Potentially, path exists already (and was simply passed as relative)
+    if not os.path.isfile(path):
```

Comment:
I think we should use `os.path.exists`. I have one entry in my database which is a series of lectures. The "file" field points to a directory with the slides. I think it's nice not to enforce the "file" field to be a single file. What do you say?

```suggestion
    if not os.path.exists(path):
```